pkgs/top-level/all-packages.nix

```diff
@@ -3589,6 +3589,8 @@ in
 
   genimage = callPackage ../tools/filesystems/genimage { };
 
+  geonkick = callPackage ../applications/audio/geonkick {};
```

Comment:
Can we please split these into separate commits?

Reply:
Geonkick is the only program in existence that uses redkite.
Does that change your opinion?
If not, I'll happily split them up.

Reply:
NM, I split them up anyway.
Thanks for the review!